.github/workflows/development-environment.yml

```diff
@@ -37,43 +36,76 @@ jobs:
 
       # Trick for unattended Docker installations
       # https://github.com/docker/for-mac/issues/2359#issuecomment-943131345
-      - name: Install prerequisites
+      # NOTE: This can sometimes take up to 10 minutes
+      - name: Install Docker
         run: |
-          HOMEBREW_NO_AUTO_UPDATE=1 brew install --cask docker
+          HOMEBREW_NO_AUTO_UPDATE=1 brew install -v --cask docker
           sudo /Applications/Docker.app/Contents/MacOS/Docker --unattended --install-privileged-components
           open -a /Applications/Docker.app --args --unattended --accept-license
-          make prerequisites
 
-      - name: Setup Python
-        uses: ./.github/actions/setup-python
+      - name: Install missing brew packages
+        run: |
+          HOMEBREW_NO_AUTO_UPDATE=1 brew install -v libxmlsec1
+
+      - name: Install volta
+        run: |
+          curl https://get.volta.sh | bash
 
-      - name: Cache (pyenv)
-        uses: actions/cache@v2
+      # This handles Python's cache
+      - name: Setup Python & cache
+        uses: ./.github/actions/setup-python
         with:
-          path: ~/.pyenv
-          key: devenv-${{ matrix.os }}-pyenv-${{ hashFiles('.python-version') }}
+          cache-files-hash: ${{ hashFiles('**/requirements.txt') }}
 
       - name: Cache (yarn)
         uses: actions/cache@v1 # We are explicitly using v1 due to perf reasons
         with:
           path: ${{ steps.info.outputs.yarn-cache-dir }}
-          key: devenv-${{ matrix.os }}-v1-yarn-${{ hashFiles('yarn.lock') }}
+          key: devenv-${{ runner.os }}-v1-yarn-${{ hashFiles('yarn.lock') }}
```

Comment:
Would it be worthwhile to keep the matrix? In the future we may want to test next version of macos before we upgrade on engineering machines.

Reply:
We can always add it back when we do.
It's far more likely we will have an OS being used by engineers before GH will make that OS available to us.

Also, the UI on GH is a bit nicer rather than a Matrix block for just 1 check.